installer/roles/create-mobile-client-crd/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+# tasks file for create-mobile-client-crd
+
+- name: Copy CRD definition to tmp file
+  template:
+    src: ../files/mobile-client-crd.yaml
+    dest: /tmp/mobile-client-crd.yaml  
+
+- name: Create mobileClient CRD
+  shell: "oc create -f /tmp/mobile-client-crd.yaml"
+
+- block:
+  - name: Get admin clusterrole
+    shell: "oc get clusterrole admin -o yaml"
```

Comment:
Is it possible to use `oc patch` here instead?

Reply:
Could use oc patch, they use replace in openshift-ansible so I thought best to use the same idea. ( https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_service_catalog/tasks/install.yml#L96 ) Also I hadn't used the replace command yet so took the opportunity to explore it. It would be fairly straightforward to switch to using oc patch.